packages/auth/e2e/user.e2e.js

```diff
@@ -154,6 +154,160 @@ describe('auth().currentUser', () => {
   });
 
   describe('sendEmailVerification()', () => {
+    it('should error if actionCodeSettings.url is not present', async () => {
```

Comment:
This line in all the new below tests: `await firebase.auth().currentUser.delete();` should be outside/below the catch block, otherwise the user won't get deleted when any of the tests fail don't throw an error